src/sagas/reown.js

```diff
@@ -130,12 +130,15 @@ function* isReownEnabled() {
 }
 
 function* init() {
-  const walletStartState = yield select((state) => state.walletStartState);
-
-  if (walletStartState !== WALLET_STATUS.READY) {
```

Comment:
chore: We can remove `WALLET_STATUS` from the imports (line 75) since we removed the only places where it was being read.

Reply:
Fixed, thanks